consensus/core/src/block.rs

```diff
@@ -50,16 +50,31 @@ impl Transaction {
     }
 }
 
+/// Index of a transaction in a block.
+pub type TransactionIndex = u16;
+
+/// Votes on transactions in a specific block.
+/// Reject votes are explicit. The rest of transactions in the block receive implicit accept votes.
+// TODO: look into making fields `pub`.
+#[derive(Clone, Deserialize, Serialize)]
+pub(crate) struct BlockTransactionVotes {
```

Comment:
Just `TransactionVotes` seems cleaner and we don't have conflict with other transaction votes outside of this

Reply:
I think it is useful to indicate the votes are for a particular Block. Let's see if the prefix will turn out to be redundant.